MantidQt/CustomInterfaces/src/Reflectometry/QDataProcessorWidget.cpp

```diff
@@ -74,7 +74,7 @@ this method is intended to be called by the presenter
 */
 void QDataProcessorWidget::setModel(const std::string &name) {
   m_toOpen = name;
-  m_presenter->notify(DataProcessorPresenter::OpenTableFlag);
+  m_presenter->notify(DataProcessorPresenter::TableUpdatedFlag);
```

Comment:
Removing this line should be enough to fix the problem. This way we can keep the original `OpenTableFlag` in the first `setModel()`